ionoscloud/resource_nfs_cluster.go

```diff
@@ -77,9 +79,10 @@ func resourceNFSCluster() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"min_version": {
-							Description: "The minimum Network File Storage version",
-							Type:        schema.TypeString,
-							Optional:    true,
+							Description:      "The minimum Network File Storage version",
+							Type:             schema.TypeString,
+							Optional:         true,
+							ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice([]string{"4.2"}, false)),
```

Comment:
I wouldn't use a validation function here. If the API adds support for another version we would have to manually add it in the code and release a new tf version.

I think the API should handle validation here.

Reply:
Removed it after all.